support-frontend/assets/helpers/tracking/thirdPartyTrackingConsent.js

```diff
@@ -1,71 +1,104 @@
 // @flow
-
-import { get as getCookie, set as setCookie } from '../cookie';
 import { logException } from 'helpers/logger';
-import { ccpaEnabled } from 'helpers/tracking/ccpa';
 import { getGlobal } from 'helpers/globals';
 
-const ConsentCookieName = 'GU_TK';
-const DaysToLive = 30 * 18;
+type ConsentVector = {
+    [key: string]: boolean;
+}
 
-const OptedIn: 'OptedIn' = 'OptedIn';
-const OptedOut: 'OptedOut' = 'OptedOut';
-const Unset: 'Unset' = 'Unset';
+type ConsentState = {
```

Comment:
will look into how we can generate flow types too...

Reply:
(in another release)

Reply:
thanks 👍